examples/mnist/train.py

```diff
@@ -68,12 +69,8 @@ def onehot(labels, num_classes=10):
   return x.astype(jnp.float32)
 
 
-def cross_entropy_loss(logits, labels):
-  return -jnp.mean(jnp.sum(onehot(labels) * logits, axis=-1))
-
-
 def compute_metrics(logits, labels):
-  loss = cross_entropy_loss(logits, labels)
+  loss = jnp.mean(optax.softmax_cross_entropy(logits, onehot(labels)))
```

Comment:
Could you also use keyword arguments here?

Reply:
Done